md/src/main/java/io/codelex/joiner/Joiner.java

```diff
@@ -0,0 +1,23 @@
+package io.codelex.joiner;
+
+import java.util.List;
+import java.util.function.BiFunction;
+
+public class Joiner <T> {
+
+    private List<T> items;
+    private BiFunction <String, T, String> function;
+
+
+    public Joiner(String separator, BiFunction<String, T, String> function) {
+        this.function = function;
+    }
+
+    public String join() {
```

Comment:
Man izskatās, ka šis īsti nestrādās, jo items vienmēr būs un paliks tukšs. 
Vai nu padod items sarakstu šai metodei, vai arī šī metode var saņemt varargs jebšu T... items